assets/src/edit-story/components/library/panes/media/common/styles.js

```diff
@@ -71,8 +71,14 @@ export const MediaGalleryLoadingPill = styled.div`
 
 export const MediaGalleryMessage = styled.div`
   color: ${({ theme }) => theme.colors.fg.white};
-  font-size: 16px;
   padding: 1em;
+  font-family: ${({ theme }) => theme.fonts.mediaError.family};
```

Comment:
Have you checked to see what this looks like for locally uploaded media?

Reply:
![Screen Shot 2020-08-21 at 10 28 37 AM](https://user-images.githubusercontent.com/13947816/90838892-6d365480-e399-11ea-8f67-aa733e516bc8.png)

Here's what it looked like for locally uploaded media.

Reply:
The top padding there like somewhat tiny compared to the media3p pane.